src/common/formUtils.ts

```diff
@@ -176,6 +188,18 @@ export const commonYupValidation = {
       }`
     );
   },
+  copyrights: Yup.string().max(
+    MAX_CHARACTER_COUNT,
+    `Must be ${MAX_CHARACTER_COUNT} characters or less`
+  ),
+  userIpi: Yup.string().max(
+    MAX_CHARACTER_COUNT,
+    `Must be ${MAX_CHARACTER_COUNT} characters or less`
+  ),
+  iswc: Yup.string().max(
+    MAX_CHARACTER_COUNT,
+    `Must be ${MAX_CHARACTER_COUNT} characters or less`
+  ),
```

Comment:
Wouldn't `IPI` and `ISWC` have more specific regex validation and not a specific character limit?